src/WebUI/Components/DeploymentsComponent.tsx

```diff
@@ -38,9 +39,9 @@ export class DeploymentsComponent extends BaseComponent<IDeploymentsComponentPro
         );
     }
 
-    private _openDeploymentItem = (item?: any, index?: number, ev?: Event) => {
-        if (this.props.onItemInvoked) {
-            this.props.onItemInvoked(item, index, ev);
+    private _openDeploymentItem = (event: React.SyntheticEvent<HTMLElement>, tableRow: ITableRow<any>) => {
+        if (this.props.onItemActivated) {
+            this.props.onItemActivated(event, tableRow);

```

Comment:
why not send the data of the selected item also? so that we do not need to worry about state